src/dso.scss

```diff
@@ -17,17 +17,19 @@ $library: true !default;
 @import "styles/mixins/bootstrap-variables";
 
 /*!
- * Bootstrap v3.3.7 (http://getbootstrap.com)
- * Copyright 2011-2016 Twitter, Inc.
- * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
- */
+* Bootstrap v3.3.7 (http://getbootstrap.com)
+* Copyright 2011-2016 Twitter, Inc.
+* Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+*/
 
 // Core variables and mixins
 @import "bootstrap-sass/assets/stylesheets/bootstrap/variables";
-@import "bootstrap-sass/assets/stylesheets/bootstrap/mixins";
+// @import "bootstrap-sass/assets/stylesheets/bootstrap/mixins";
+@import "styles/mixins/bootstrap-mixins";
```

Comment:
Zullen we de bootstrap mixins direct in deze `dso-scss` file zetten? Ja, de file wordt dan 45 regels langer maar het hele doel van deze file is de style-omgeving optuigen.

Het is heel raar dat er vanuit een `styles` directory naar een hele rits Bootstrap resources wordt verwezen

Reply:
ok